specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/readme.typescript.md

```diff
@@ -6,7 +6,7 @@ Please also specify `--typescript-sdks-folder=<path to root folder of your azure
 ``` yaml $(typescript)
 typescript:
   azure-arm: true
-  package-name: "@azure/arm-hdinsightonaks"
-  output-folder: "$(typescript-sdks-folder)/sdk/hdinsight/arm-hdinsightonaks"
+  package-name: "@azure/arm-hdinsightcontainers"
```

Comment:
Could you also suggest a title for typescript?

Reply:
Hi @MaryGao I thought the js is the same with typescript. If it is not, could you please help us generate one, as our namespace is hdinsightcontainers now? Per the approved issue https://github.com/Azure/azure-sdk-pr/issues/1247

Reply:
Please use "HDInsightContainersManagementClient" as name to keep the same style with other azure services